pom.xml

```diff
@@ -1,122 +1,218 @@
+  <modelVersion>4.0.0</modelVersion>
 
-    <groupId>com.epam.tc</groupId>
-    <artifactId>Dmitrii_Pikhurov</artifactId>
-    <version>1.0-SNAPSHOT</version>
+  <groupId>com.epam.tc</groupId>
+  <artifactId>Dmitrii_Pikhurov</artifactId>
+  <version>1.0-SNAPSHOT</version>
 
 
-    <properties>
-        <maven.compiler.source>11</maven.compiler.source>
-        <maven.compiler.target>11</maven.compiler.target>
-        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <testng.version>7.4.0</testng.version>
-        <assertj-core.version>3.20.2</assertj-core.version>
-        <maven-surefire-plugin.version>3.0.0-M5</maven-surefire-plugin.version>
-        <maven-surefire-plugin.suiteXmlFile>${project.build.testOutputDirectory}/testng/calculator.homework1.addition.subtraction.xml
-        </maven-surefire-plugin.suiteXmlFile>
-        <calculator.version>1</calculator.version>
-        <webdrivermanager.version>4.4.3</webdrivermanager.version>
-        <selenium-java.version>4.0.0-beta-4</selenium-java.version>
-    </properties>
+  <properties>
+    <maven.compiler.source>11</maven.compiler.source>
+    <maven.compiler.target>11</maven.compiler.target>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <testng.version>7.4.0</testng.version>
+    <assertj-core.version>3.20.2</assertj-core.version>
+    <maven-surefire-plugin.version>3.0.0-M5</maven-surefire-plugin.version>
+    <maven-surefire-plugin.suiteXmlFile>
+      ${project.build.testOutputDirectory}/testng/calculator.homework1.addition.subtraction.xml
+    </maven-surefire-plugin.suiteXmlFile>
+    <calculator.version>1</calculator.version>
+    <webdrivermanager.version>4.4.3</webdrivermanager.version>
+    <selenium-java.version>4.0.0-beta-4</selenium-java.version>
+    <allure.version>2.13.2</allure.version>
+    <aspectj.version>1.9.5</aspectj.version>
+    <allure-maven.version>2.10.0</allure-maven.version>
+  </properties>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <version>${maven-surefire-plugin.version}</version>
+        <configuration>
+          <argLine>
+            --illegal-access=permit
+            -javaagent:"${settings.localRepository}/org/aspectj/aspectjweaver/${aspectj.version}/aspectjweaver-${aspectj.version}.jar"
+          </argLine>
+          <suiteXmlFiles>
+            <suiteXmlFile>
+              ${maven-surefire-plugin.suiteXmlFile}
+            </suiteXmlFile>
+          </suiteXmlFiles>
+        </configuration>
+        <dependencies>
+          <dependency>
+            <groupId>org.aspectj</groupId>
+            <artifactId>aspectjweaver</artifactId>
+            <version>${aspectj.version}</version>
+          </dependency>
+        </dependencies>
+      </plugin>
+
+      <plugin>
+        <groupId>com.alexecollins.maven.plugin</groupId>
```

Comment:
Why exactly do yo need this plugin?

Reply:
It executes JavaScript script at site phase and opens allure report in browser automatically. Vladimir on lecture did it manually.

Reply:
Script on 68 line.